### PR TITLE
upgrade fabric8.version to 2.2.193, includes:

### PR DESCRIPTION
**Pull request by rawlingsj**
- Rename of PetSets to StatefulSets
 - CronJobs
 - Resolves breaking changes from the kubernetes model and client major version upgrade

### CODE REVIEW

**Comment by rawlingsj** [comment]
@rhuss would you be able to give this a quick review when you get a min please?

File: core/src/main/java/io/fabric8/maven/core/util/KubernetesResourceUtil.java [hunk 1]
```diff
@@ -197,6 +197,7 @@ public boolean accept(File dir, String name) {
             // lets put the abbreviation we want to use first
             "cm", "ConfigMap",
             "configmap", "ConfigMap",
+            "cj", "CronJob",
```
Comment (jimmidyson) [inline core/src/main/java/io/fabric8/maven/core/util/KubernetesResourceUtil.java]: Can we add `cronjob` as well as `cj` as we have for `configmap` and `cm` above?

Comment (rawlingsj) [inline core/src/main/java/io/fabric8/maven/core/util/KubernetesResourceUtil.java]: yeah good call - added.

**Comment by rawlingsj** [comment]
I'll just rebase this PR too

**Comment by rhuss** [comment]
lgtm

**Comment by rawlingsj** [comment]
Thanks @rhuss @jimmidyson - what's the process now?  Can this be merged?  Once done I'll release and update fabric8-devops and add in our new CronJob for auto updates on fabric8 :)

**Comment by jimmidyson** [comment]
LGTM [merge]

**Pull request merged**